### PR TITLE
add docker file

### PR DESCRIPTION
**Pull request by swflb**

### CODE REVIEW

**Comment by swflb** [comment]
@incessantier @bnhhughes just adding the docker file so I can begin using a docker and create some auto builds

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM gcc:4.9.4
+# assumes this alread has git installed
+#RUN apt-get -y update && apt-get install -y git
+RUN mkdir /opt/externpro
+COPY externpro-16.01.1-gcc494-64-Linux.sh /opt/externpro/
+RUN cd /opt/externpro
+RUN chmod +x /opt/externpro/externpro-16.01.1-gcc494-64-Linux.sh
+RUN yes | /opt/externpro/externpro-16.01.1-gcc494-64-Linux.sh --prefix=/opt/externpro
+RUN dpkg -i /opt/externpro/externpro-16.01.1-gcc494-64-Linux/pkg/cmake-3.3.2-Linux-x86_64.deb
```
Comment (bnhhughes) [inline Dockerfile:9]: I take it you're running this on ubuntu and not centos? The other ovsrpro stuff supports RPM but not DEB packages.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+#CMD cmake --version && git --version && ls -al /opt/externpro/
+CMD cd /tmp
+CMD git clone --recurse-submodules https://github.com/distributePro/ovsrpro.git
```
Comment (bnhhughes) [inline Dockerfile:13]: I suppose it doesn't hurt anything, but the `--recurse-submodules` isn't necessary - there's no submodules in ovsrpro

File: Dockerfile (hunk 1 above)
Comment (swflb) [reply Dockerfile:9]: Yes. Debian based. Note also the GCC version of 4.9.4 instead of 4.9.2 like in devtoolset3

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,21 @@
+FROM gcc:4.9.4
+# assumes this alread has git installed
+#RUN apt-get -y update && apt-get install -y git
+RUN mkdir /opt/externpro
+COPY externpro-16.01.1-gcc494-64-Linux.sh /opt/externpro/
```
Comment (bnhhughes) [inline Dockerfile:5]: where does externpro have to live?  You could clone it and build it the same as is done with ovsrpro if you wanted...

Comment (swflb) [reply Dockerfile:5]: It has to live in the folder in which the docker is created. I used this to create a docker image and of that up

**Pull request merged**